app/assets/javascripts/respondent_detail.es6

```diff
@@ -0,0 +1,31 @@
+$(document).ready(() => {
+  if ($("#bail_conditions_set").length) {
+    const bailConditionOptions = "input[type = radio][name = 'respondent[bail_conditions_set]']";
+    $(bailConditionOptions).on("change", setBailConditionsLabel);
```

Comment:
Wouldn't it be useful to also change the error message?
```javascript
    $(bailConditionOptions).on("change", () => {
      setBailConditionsLabel()
      updateError()
    })
```

Reply:
Good suggestion. I was just apprehensive to add that (changing error message) at the time.